### ISSUE TITLE
YouTube-like scroll down in fullscreen mode

### ISSUE THREAD

**Issue by GilgusMaximus** [FreeTubeApp/FreeTube#887]
**Is your feature request related to a usage problem (not a bug)? Please describe.**
No.

**Describe the solution you'd like to see implemented**
On YouTube, you can scroll down from a video, even when it is in full screen. It remains in full screen, and you can scroll back up to have the video display like normal full screen. This would make it easier to quickly look into the description, or into the comments, without the need to close fullscreen. This probably requires some heavier tweaking in the page and videoplayer structure.

**Screenshots**
![image](https://user-images.githubusercontent.com/34301369/102688602-17af6e80-41f8-11eb-875f-86c5b453396a.png)
Example of how it looks when you scroll down in fullscreen

**Comment by kubo6472** [comment]
This is definitely one of the features I hate the most on YouTube. Could this be implemented along with a setting option?

**Comment by shivasagarrao** [comment]
If this feature is implemented make a setting to turn off.

**Comment by GilgusMaximus** [comment]
Of course, this will become a setting if implemented. However at the moment I am not so sure how this would be implemented at all, so I'll just leave it here for anyone to checkout

**Comment by pseudomonas** [comment]
Note that #4364 was requesting this feature for full **window** not full-**screen**.
